content/en/platform/corda/5.1/application-networks/creating/members/reregister.md

```diff
@@ -172,7 +172,7 @@ You can learn more about configuring the registration process in the [Managing M
    {{% /tab %}}
    {{< /tabs >}}
 
-4. Send a re-registration request using the common registration/re-registration endpoint.
+4. Send a re-registration request using the common [registration/re-registration endpoint](https://docs.r3.com/en/platform/corda/5.0/reference/rest-api/openapi.html#tag/Member-Registration-API/operation/post_membership__holdingidentityshorthash_).
```

Comment:
I think you should be able to specify this as a relative link.. probably ../../../reference/rest-api/openapi.html#tag/Member-Registration-API/operation/post_membership__holdingidentityshorthash_

Reply:
updated